### PR TITLE
Add v0.6.0 of mattermost-plugin-calls to the Marketplace

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- cut with `/mb cutplugin --repo mattermost-plugin-calls --tag v0.6.0--pre-release`
- this commit made with `go run ./cmd/generator/ add mattermost-plugin-calls v0.6.0 --official --beta --on-prem`

#### Ticket Link
- none

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -10379,23 +10482,23 @@
...
-    "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-zoom-v1.5.1.tar.gz",
-    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-zoom/releases/tag/v1.5.1",
+    "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-zoom-v1.6.0.tar.gz",
```
Comment (lieut-data) [inline plugins.json:10485]: Why is the Zoom plugin changing?

**Review by lieut-data — CHANGES_REQUESTED** [review]

File: plugins.json [hunk 2]
```diff
@@ -10499,40 +10602,40 @@
...
-    "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-zoom-v1.6.0.tar.gz",
-    "release_notes_url": "https://github.com/mattermost/mattermost-plugin-zoom/releases/tag/v1.6.0",
+    "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-zoom-v1.5.1.tar.gz",
```
Comment (streamer45) [inline plugins.json:10622]: Is this expected?

File: plugins.json (hunk 1 above)
Comment (cpoile) [reply plugins.json:10485]: @lieut-data I have no idea how that got in there. It wasn't in my local diff...

**Comment by cpoile** [comment]
The generator is broken for this one, it keeps updating zoom as well. I haven't taken the time to look at the code, though. If it happens again I will.

**Comment by streamer45** [comment]
@cpoile We could also hold off this PR if you are worried. I don't think there's any hard deadline for this other than June 15th.

**Comment by lieut-data** [comment]
> The generator is broken for this one, it keeps updating zoom as well. I haven't taken the time to look at the code, though. If it happens again I will. I kind of want to figure out what's going on, but I'm afraid it will take up too much time...

I think it would be fine to just revert this portion of the changes and use only the part re: calls. We aren't required to pull in all the new marketplace changes.

**Comment by cpoile** [comment]
@streamer45 I think we're good to go, not too worried about it. Just wondering what happened. :)

File: plugins.json [hunk 3]
```diff
@@ -3300,6 +3300,109 @@
...
+            "placeholder": "stun:example.com:3478",
+            "default": "stun:stun.global.calls.mattermost.com:3478"
+          },
+          {
+            "key": "RTCDServiceURL",
+            "display_name": "RTCD service URL",
+            "type": "text",
+            "help_text": "(Optional) The URL to a running RTCD service instance that should host the calls. When set (non empty) all calls will be handled by the external service.",
+            "placeholder": "https://rtcd.example.com",
+            "default": null
```
Comment (streamer45) [inline plugins.json:3391]: Hope this won't cause issues, it shouldn't really.

**Review by lieut-data — APPROVED** [review]

File: plugins.json (hunk 3 above)
Comment (cpoile) [reply plugins.json:3391]: Ah, it's because we don't have a `default` specified. Up to you, we could manually change it here then change it in code later.

**Review by streamer45 — APPROVED** [review]

**Pull request merged**